src/pages/List.js

```diff
@@ -299,6 +300,10 @@ export default function List({ token }) {
           {listItems.docs.length === 0 ? (
             <section className="flex flex-col items-center w-full">
               <p>Your grocery list is currently empty.</p>
+              <img
+                src={writingToken}
+                alt="hand holding pen to write on paper"
+              />
```

Comment:
Image looks great!

Our Figma design has the `<p>` below the image, and it reads: "You don’t have any listed items."

And I think some Tailwind utility classes to add spacing will help visually! So I'm imagining something like:

```javascript
<img
  src={writingToken}
  alt="hand holding pen to write on paper"
  className="mb-5"
/>
 <p className="mb-5">You don’t have any listed items.</p>
```

Reply:
I was going to hold off doing much (if any) alignment since we don't have the final verbiage and font sizes in yet, and it is covered in another PR. I did add the margin like you suggested, and started to experiment with columns, but then decided to get this pushed with images to hopefully help others along on their PRs  👍

Reply:
Fair point!!

So everything looks and works great!

That said, I am honestly not a big fan of the image quality on the desktop, when it's so big! It's not very sharp.

BUT. I Think for now we just move on and if we can come back to this, awesome. If not, no big deal.

Reply:
APPROVED!